menas/src/main/resources/envoy/start_envoy.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+mkdir -p /etc/ssl
+
+echo -e "$PRIVATE_KEY" >> /etc/ssl/envoy.key
+echo -e "$CERTIFICATE" >> /etc/ssl/envoy.crt
+echo -e "$CA_CHAIN" >> /etc/ssl/envoy.crt
```

Comment:
As per @hamc17 's previous suggestion

```suggestion
echo -e "${PRIVATE_KEY}" >> /etc/ssl/envoy.key
echo -e "${CERTIFICATE}" >> /etc/ssl/envoy.crt
echo -e "${CA_CHAIN}" >> /etc/ssl/envoy.crt
```